### DISCUSSION TITLE
Why use :where(html)?

### DISCUSSION THREAD

**Discussion by sanbornhilland** [argyleink/open-props#332] [Q&A] [answered]
I think open-props is an awesome library/resource. Recently I've been poking around in the source to pick up whatever tips and tricks I can and I noticed that all properties are declared on the selector `:where(html)`. It's not immediately clear to me why that is and I haven't found a specific explanation anywhere else. I've also typically seen props declared on `:root`. I am assuming this decision ultimately comes down to a decision over specificity but if you have a few free minutes I'd really love to see a clear explanation of why you would use `:where(html)` rather than `html` or `:root`. 

🍻 again for putting out such a great library.

**Comment by argyleink** [comment, marked as answer]
Hi! Thanks for checkin in 🙂 

Everything is wrapped in `:where()` so you as a consumer of the library won't ever have to battle the specificity to overwrite things with your own custom values. The props or normalize load in and don't require any battles, making itself weak instead of strong with specificity. Same with the choice for `:where(html)`, it's just weaker in intention, like good faith towards this being helpful and not intrusive. Really it could be `:where(:root)` and nothing would change, but yeah, it's the intention of being a subservient library. 

Glad you're diggin the lib! <3

**Reply by sanbornhilland** [reply]
Awesome thanks for the quick reply!
